src/main/java/com/iexec/worker/dataset/DataService.java

```diff
@@ -55,6 +55,12 @@ public boolean downloadFiles(String chainTaskId, List<String> uris) {
         return true;
     }
 
+    public boolean unzipDownloadedTeeDataset(String chainTaskId, String datasetUri) {
```

Comment:
Maybe we should check the `datasetUri` emptiness otherwise `.getFileName()` could be `null`.